figures/plot_settings.jl

```diff
@@ -1,5 +1,4 @@
 # PLOT SETTINGS
-textwidth = 469.75
```

Comment:
This variable is the width of the text in the Latex doc. It is so the user can appropriately size figures to fit on the page (and therefore allow figure text to match document text in size).

It is already used in generating the figures in `synthetic_comparison.ipynb`. Having it here allows all plots to be generated with the same parameters. We could rename it to `doc_textwidth` or something of the sort.

Reply:
ah got it. Renamed it to doc_textwidth now. We'll need to change that line in `synthetic_comparison.ipynb` next time we run it. Good to merge for now?